benchmarks/htap/lib/controller.py

```diff
@@ -71,6 +71,13 @@ def olap_worker(self, worker_id):
         while True:
             queries.run_next_query()
 
+    def analyze_worker(self):
+        tables = ['customer', 'district', 'history', 'item', 'nation', 'new_orders', 'order_line', 'orders', 'region', 'stock', 'supplier', 'warehouse']
```

Comment:
Please wrap lines to reasonable length, I prefer 80, but I guess 100 or so is good as well.

Reply:
done